bigbluebutton/greenlight
# add(feature): SuperAdmin [1]

- Add Admin Panel - Manage Tenants layout
- Add Tenant#create

How To Test:
Set your user provider to "bn" `my_user.provider = "bn"`
Set your user role to "SuperAdmin" `my_user.role_id = Role.find_by(name: 'SuperAdmin').id`
Save your user.  `my_user.save`

You should now have access to "AdminPanel/ManageTenants"

Go to ManageTenants
Create a new Tenant
Verify in the console that the new tenant has been created.
